packages/lib/src/vis/shared/Annotation.tsx

```diff
@@ -10,18 +11,30 @@ interface Props extends HTMLAttributes<HTMLDivElement> {
   x: number;
   y: number;
   scaleOnZoom?: boolean;
+  center?: boolean;
 }
 
 function Annotation(props: Props) {
-  const { x, y, scaleOnZoom, children, style, ...divProps } = props;
+  const { x, y, scaleOnZoom, center, children, style, ...divProps } = props;
 
   const camera = useThree((state) => state.camera);
+  useFrameRendering();
 
   const { dataToWorld, worldToHtml } = useAxisSystemContext();
   const htmlPt = worldToHtml(dataToWorld(new Vector2(x, y)));
 
-  useFrameRendering();
-
+  if ((center || scaleOnZoom) && style?.transform) {
```

Comment:
Perhaps add `transformOrigin` to the check ?

Reply:
My feeling was that one would not use one without the other. :shrug:

Reply:
If someone uses `transform-origin` on its own, it'd probably be to override our own value on purpose, and I didn't want to prevent that.